### DISCUSSION TITLE
False positive detection

### DISCUSSION THREAD

**Discussion by MetaSamsara** [winsiderss/systeminformer#824] [Questions / Answers] [answered]
Hi, I saw on the website forum that people have reported false positive detections before, but I found none recent so I figured I would let you know about it. Firefox (version 86.0) tells me the download for `processhacker-3.0.3866-setup.exe` contains a virus.

[CF this screenshot](https://i.imgur.com/74tXk8P.png)

**Comment by Biswa96** [comment, marked as answer]
> but I found none recent 

C'mon! We're chilling here https://github.com/processhacker/processhacker/issues/454. Join the Anti-virus ranting party 🥳
